pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/EtterlatteBrev.kt

```diff
@@ -58,7 +59,6 @@ data class AvkortetBeregningsperiode(
 data class Utbetalingsinfo(
     val antallBarn: Int,
     val beloep: Kroner,
-    val grunnbeloep: Kroner,
```

Comment:
Vil ikke grunnbeløp nå bli utilgjengelig for barnepensjon? 🤔

Reply:
Vil være tilgjengelig for barnepensjon gjennom beregningsperioder. Det feltet ble kun brukt av OMS